src/uu/uniq/uniq.md

```diff
@@ -0,0 +1,15 @@
+# uniq
+
+```
+uniq [OPTION]... [INPUT [OUTPUT]]...
+```
+
+Report or omit repeated lines.
+
+## After help
+
+Filter adjacent matching lines from `INPUT` (or standard input),
+writing to `OUTPUT` (or standard output).
+
+Note: 'uniq' does not detect repeated lines unless they are adjacent.
+You may want to sort the input first, or use 'sort -u' without 'uniq'.
```

Comment:
```suggestion
Note: `uniq` does not detect repeated lines unless they are adjacent.
You may want to sort the input first, or use `sort -u` without `uniq`.
```

One small suggestion, but looks good!

Reply:
But the help message in command line might looks like the following with ` removed:

```diff
-Note: 'uniq' does not detect repeated lines unless they are adjacent.
-You may want to sort the input first, or use 'sort -u' without 'uniq'.
+Note: uniq does not detect repeated lines unless they are adjacent.
+You may want to sort the input first, or use sort -u without uniq.
```